README.md

```diff
@@ -321,7 +318,6 @@ other package management tools such as `apt-get`:
 [6]: https://clang.llvm.org/docs/ClangFormatStyleOptions.html
 [7]: https://cibuildwheel.readthedocs.io/en/stable/
 [8]: https://clang.llvm.org/extra/clang-tidy/
-[9]: https://github.com/rizsotto/Bear
```

Comment:
Do we want to reshuffle the indices here?

Reply:
sigh, good point... I will reuse this index [9] for `gersimi` (which I forgot to add)